store/src/java/com/zimbra/cs/lmtpserver/ExternalEmailWarning.java

```diff
@@ -153,6 +152,15 @@ public boolean isExternal(String receiverAddress, String originatorAddress) {
         return false;
     }
 
+    private static final String CONTENT_TRANSFER_BASE64_TEXT_PLAIN_REGEX =
+            "Content-Transfer-Encoding: base64(?:(?!\\r\\n\\r\\n).)*?Content-Type: text/plain(?:(?!\\r\\n\\r\\n).)*?\\r\\n\\r\\n";
+    private static final Pattern CONTENT_TRANSFER_BASE64_TEXT_PLAIN_PATTERN = Pattern.compile(CONTENT_TRANSFER_BASE64_TEXT_PLAIN_REGEX,
+            Pattern.CASE_INSENSITIVE + Pattern.DOTALL);
+    private static final String CONTENT_TRANSFER_BASE64_TEXT_HTML_REGEX =
+            "Content-Transfer-Encoding: base64(?:(?!\\r\\n\\r\\n).)*?Content-Type: text/html(?:(?!\\r\\n\\r\\n).)*?\\r\\n\\r\\n";
```

Comment:
declare a variable for Content-Transfer-Encoding: base64

Reply:
Changes done as suggested